base/src/org/compiere/model/MInOut.java

```diff
@@ -1111,10 +1111,15 @@ public String prepareIt()
 		//	Credit Check
 		if (isSOTrx() && !isReversal())
 		{
-			I_C_Order order = getC_Order();
+			MOrder order = (MOrder)getC_Order();
+			Boolean IsApprovedForDelivery = false;
+			if (order !=null)
+				IsApprovedForDelivery = order.get_ValueAsBoolean("IsApprovedFordelivery");
```

Comment:
Please update your branch and generate X_ and I_ class for use constants from model class of Sales Orders
suggested:
```Java
isApprovedForDelivery = order.isApprovedForDelivery();
```